php/class-photonfill.php

```diff
@@ -688,14 +681,47 @@ public function get_attachment_picture( $attachment_id, $size = 'full', $attr )
 						}
 
 						// Set our default img element
-						$html .= '<img srcset="' . esc_url( $default_srcset ) . '"' . $alt . ' ' . $style . '>';
+						$attr['srcset'] = $default_srcset;
+						$html .= $this->build_attachment_image( $attachment_id, $attr );
 						$html .= '</picture>';
 					}
 				}
 				return $html;
 			}
 			return;
 		}
+		
+		/**
+		 * Get the HTML for the IMG Tag
+		 *
+		 * @param $attachment_id int Attachment ID
+		 * @param $attr array Image Attributes
+		 *
+		 * @return string
+		 */
+		private function build_attachment_image( $attachment_id, $attr ) {
+			$attachment = get_post( $attachment_id );
+			
+			if ( empty( $attr['alt'] ) ) {
+				$attr['alt'] = trim( strip_tags( get_post_meta($attachment_id, '_wp_attachment_image_alt', true ) ) );
+			}
+			if ( empty($attr['alt'] ) ) {
```

Comment:
I can see how it looks wrong, but I think it's correct.  It's closely copied from `wp_get_attachment_image()`.  Not that they wrote it in the best way for readability, but the idea is this:

First, if alt is empty, it tries to get it from `_wp_attachment_image_alt`.  We then want to check again to see if it's still empty, and if it is, get it from the post excerpt.  Then, check it again, if it's still empty, get it from the title.  The alt tag will be the first non-empty value.